PAWS/Onboarding/InvitationCodeError.swift

```diff
@@ -11,12 +11,17 @@ import Foundation
 
 enum InvitationCodeError: LocalizedError {
     case invitationCodeInvalid
-    
+    case userNotAuthenticated
+    case generalError(String)
     
     var errorDescription: String? {
         switch self {
         case .invitationCodeInvalid:
-            String(localized: "Invitation code was invalid.")
+            NSLocalizedString("The invitation code is invalid or has already been used.", comment: "Invitation Code Invalid")
+        case .userNotAuthenticated:
+            NSLocalizedString("User authentication failed. Please try to sign in again.", comment: "User Not Authenticated")
+        case .generalError(let message):
+            String(format: NSLocalizedString("An error occurred: %@", comment: "General Error"), message)
```

Comment:
You should be able to generate a localized string like this:
```suggestion
            String(localized: ...)
```